<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>Form</class>
 <widget class="QWidget" name="Form" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>139</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Calculator</string>
  </property>
  <widget class="QWidget" name="layoutWidget" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>50</y>
     <width>351</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" >
    <property name="margin" >
     <number>0</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item>
     <widget class="QLineEdit" name="lineEdit_a" >
      <property name="inputMask" >
       <string/>
      </property>
      <property name="text" >
       <string>0</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox" >
      <item>
       <property name="text" >
        <string>+</string>
       </property>
      </item>
      <item>
       <property name="text" >
        <string>-</string>
       </property>
      </item>
      <item>
       <property name="text" >
        <string>*</string>
       </property>
      </item>
      <item>
       <property name="text" >
        <string>/</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_b" >
      <property name="inputMask" >
       <string/>
      </property>
      <property name="text" >
       <string>0</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_eauql" >
      <property name="text" >
       <string>Ôºù</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_c" >
      <property name="text" >
       <string>0</string>
      </property>
      <property name="readOnly" >
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget" >
   <property name="geometry" >
    <rect>
     <x>100</x>
     <y>100</y>
     <width>182</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" >
    <property name="margin" >
     <number>0</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item>
     <widget class="QPushButton" name="pushButton_ok" >
      <property name="text" >
       <string>Ok</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_close" >
      <property name="text" >
       <string>Close</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_compute" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>20</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>Compute:</string>
   </property>
  </widget>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_close</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>250</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel" >
     <x>380</x>
     <y>115</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
